bofire/data_models/surrogates/api.py

```diff
@@ -50,7 +55,8 @@
         SingleTaskGPSurrogate,
         MixedSingleTaskGPSurrogate,
         MixedTanimotoGPSurrogate,
-        MLPEnsemble,
+        ClassificationMLPEnsemble,
+        RegressionMLPEnsemble,
```

Comment:
Sure, right now, `AnyClassificatonSurrogate = ClassificationMLPEnsemble` will need to be sufficient since it is the only surrogate at this point in time

Reply:
Correct!